app/controllers/identities_controller.rb

```diff
@@ -20,6 +20,17 @@ def show
     render plain: "404 Not Found", status: :not_found unless @identity
   end
 
+  def toggle_portal
+    return redirect_to root_path unless admin?
+
+    form_user
+    portal = Flipper[flipper_params]
```

Comment:
would this code be a little simpler if this line was inside your switch_portal method? I don't think you need flipper_params either

Reply:
great, haven't thought about that - will try to rewrite it.
my idea of `flipper_params` was to protect from injections, but wasn't sure if it is doing anything in this case

Reply:
done, moved into before action / app controller for future use